### DISCUSSION TITLE
Change order of posts?

### DISCUSSION THREAD

**Discussion by mrashco** [adityatelange/hugo-PaperMod#1071] [Q&A]
Hi there,

By default, the order of posts is by date, so newer posts come first. Is there a way to change this?
I'd like to order posts alphabetically or numerically, for example.

Thanks,
Ash

**Comment by mrashco** [comment]
If dates are removed from each post, then they display alphabetically.

**Comment by RoneoOrg** [comment]
You can use `weight:` in the front matter
That's what I use and I don't think I modified the theme to make this work
Let me know if it's not enough

**Reply by mrashco** [reply]
Thanks, I'll try this.
